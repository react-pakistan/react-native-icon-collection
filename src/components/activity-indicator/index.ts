export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
