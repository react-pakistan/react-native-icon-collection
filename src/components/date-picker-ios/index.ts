export { DatePickerIOS } from './date-picker-ios';
