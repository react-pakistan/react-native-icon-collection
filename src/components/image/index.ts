export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
