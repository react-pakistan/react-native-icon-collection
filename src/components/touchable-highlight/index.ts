export { TouchableHighlight } from './touchable-highlight';
