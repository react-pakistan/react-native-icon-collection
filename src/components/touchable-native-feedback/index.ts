export { TouchableNativeFeedback } from './touchable-native-feedback';
