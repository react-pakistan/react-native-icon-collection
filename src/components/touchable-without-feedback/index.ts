export { TouchableWithoutFeedback } from './touchable-without-feedback';
