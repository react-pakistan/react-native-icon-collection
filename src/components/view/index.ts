export { View, IView } from './view';
