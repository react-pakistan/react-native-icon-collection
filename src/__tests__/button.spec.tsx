import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '../components/button';
import {
  buttonProps,
} from '../../__stories__/story/button.story';

describe('Testing Button component', () => {
  it('matches Button snapshot, flex column', () => {
    const wrapper = renderer.create(
      <Button {...buttonProps} />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
