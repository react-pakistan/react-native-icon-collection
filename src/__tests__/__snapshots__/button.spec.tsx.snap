// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Button component matches Button snapshot, flex column 1`] = `
<View
  accessibilityLabel="Press Me"
  accessibilityRole="button"
  accessibilityStates={Array []}
  accessible={true}
  focusable={true}
  hasTVPreferredFocus={false}
  isTVSelectable={true}
  nextFocusDown={0}
  nextFocusForward={0}
  nextFocusLeft={0}
  nextFocusRight={0}
  nextFocusUp={0}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID="testID"
>
  <View
    style={
      Array [
        Object {},
      ]
    }
  >
    <Text
      disabled={false}
      style={
        Array [
          Object {
            "color": "#007AFF",
            "fontSize": 18,
            "padding": 8,
            "textAlign": "center",
          },
          Object {
            "color": "#00D3CA",
          },
        ]
      }
    >
      Press Me
    </Text>
  </View>
</View>
`;
